server/bin/www

```diff
@@ -1,8 +1,5 @@
 #!/usr/bin/env node
 
-/* Sets up the environment variables from your .env file*/
-require("dotenv").config();
-
```

Comment:
I think we still want this right?

Reply:
Everything runs with that line being in app.js instead, I think it makes more sense in there anyway since that's where all the express configuration is happening.